nautobot/core/templates/generic/object_bulk_import.html

```diff
@@ -88,13 +88,7 @@ <h1>{% block title %}{{ obj_type|bettertitle }} Bulk Import{% endblock %}</h1>
                                 <td>
                                     <code>{{ name }}</code>
                                 </td>
-                                <td>
-                                    {% if field.required %}
-                                        <i class="mdi mdi-check-bold text-success" title="Required"></i>
-                                    {% else %}
-                                        <span class="text-muted">&mdash;</span>
-                                    {% endif %}
-                                </td>
+                                <td>{{ field.required | render_boolean }}</td>
```

Comment:
This is one of the few user-noticeable rendering changes in this PR, I think? Since now non-required fields are being indicated with a :x: instead of a —? I think I'm OK with this change, personally.